cmd/event_loop.go

```diff
@@ -81,7 +81,7 @@ func eventLoop(workerErrs <-chan error, signals <-chan os.Signal, subscription *
 func setupUI(unsubscribe func() error, ux ui.UI) (ui.UI, error) {
 	if err := ux.Setup(unsubscribe); err != nil {
 		// replace the existing UI with a (simpler) logger UI
-		ux = ui.NewLoggerUI()
+		ux = ui.NewLoggerUI(os.Stdout)
```

Comment:
Want to make sure I follow: Why `os.Stdout` here?

Reply:
Disregard the above — in favor of this comment -> https://github.com/anchore/syft/pull/530#discussion_r722450745

Reply:
I've done a tiny refactor so that the caller can provide fallback UIs to use in case the preferred one does not work